.renovaterc.json

```diff
@@ -26,10 +26,11 @@
       "excludePackagePatterns": [
         "^@adobe/"
       ],
+      "pin": false,
```

Comment:
I think this is wrong: 
https://docs.renovatebot.com/configuration-options/#pin

@trieloff do you know how to do this?

Reply:
maybe this: https://docs.renovatebot.com/presets-default/#donotpinpackageltarg0gt